### PR TITLE
[FIX] My Orders: use standart combobox (RT-2963)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
Use standart combobox for My Orders mobile view sort dropdown

### CODE REVIEW

**Comment by darkdarkdragon** [comment]
before
rt-2963-before

**Comment by darkdarkdragon** [comment]
after
rt-2963-after

**Comment by rippletravis** [comment]
Visuals look good.

**Comment by mrajvanshy** [comment]
LGTM

File: src/jade/strings/myOrders.jade [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//- just string for My Orders sort combobox
```
Comment (vhpoet) [inline src/jade/strings/myOrders.jade]: why is this in a separate file?

**Comment by jublo** [comment]
The checkbox is browser- and OS-specific. Is this okay, @rippletravis?

image

**Comment by vhpoet** [comment]
The code is too complex for this. Let's just work on rp-combobox to accept options as html just like the select itself.

``` 
rp-combobox
  span option1
  span option2

**Comment by jublo** [comment]
@vhpoet Can you create a JIRA issue and assign it to me?
@tonga Can you move that issue up on my list once it's created, please?

**Comment by vhpoet** [comment]
@jublo https://ripplelabs.atlassian.net/browse/RT-3062

**Comment by jublo** [comment]
@darkdarkdragon Please rebase.

**Comment by darkdarkdragon** [comment]
@jublo done

**Comment by vhpoet** [comment]
LGTM

**Comment by vhpoet** [comment]
@jublo will you also change this code after you implement RT-3062?

**Pull request merged**